src/prefab/construct-env.ts

```diff
@@ -36,7 +36,7 @@ export default async function(pkgenv: { installations: Installation[] }) {
 
 ///////////////////////// reworked from useShellEnv needs porting back to libpkgx
 async function mkenv({installations}: {installations: Installation[]}) {
-  const projects = new Set(installations.map(x => x.pkg.project))
+  const projects = new Set(installations.map(x => `${x.pkg.project}@${x.pkg.version}`))
```

Comment:
Can't remember why we added this anyway.

Reply:
it's been a while, but i believe it's because we can allow multiple unicode lib versions in the env simultaneously, and this check trips if there are two or more (since they map to the same name).

Reply:
I seem to recall there was an issue if an env had multiple of the same exact thing but can't remember.

Reply:
i _do_ recall at one point early on we would sometimes get the pkg@ver duplicated; perhaps this was to ensure that bug never returned.